content/en/contribute/medic/product-development-process/_index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Product Development Process"
+linkTitle: "Product Development Process"
+weight: 2
+description: >
+  What Product Development Process use follow at Medic
```

Comment:
Started by fixing the typo, but then figured we could make the description more descriptive so that it shows well on the summary page.

```suggestion
  Explanation of the Continuous Discovery process used for product development at Medic
```